WebClient/src/main/java/service/home/WebApp.java

```diff
@@ -0,0 +1,16 @@
+package service.home;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class WebApp {
+
+    public static void main(String ... args){
+
+        System.setProperty("spring.config.name", "webapp-server");
```

Comment:
And the name in YAML is "webapp-app"? Sounds amazingly frosty (cause cool just won't cut it here). Rolls of my tongue. Webapp-app-app. Oh-app.